src/app/edit-recipe/edit-recipe.component.ts

```diff
@@ -12,19 +12,70 @@ export class EditRecipeComponent implements OnInit {
     ingredient: '',
     measure: ''
   }];
+
   public disabled = true;
+  public recipe;
+  public name: string = '';
+  public time;
+  public description: string;
+  public instruction: string;
+  public idMeal: number;
```

Comment:
This name kind of bugs me. Maybe rename it to `recipeId` ?

Reply:
In fact, you can create a `RecipeModel` class and use it here instead of:

```
public recipe;
  public name: string = '';
  public time;
  public description: string;
  public instruction: string;
  public idMeal: number;
public image: string = '';
```